sigstore/_internal/trust.py

```diff
@@ -386,9 +387,7 @@ def get_tsa_urls(self) -> list[str]:
         but may return more in future.
         """
         url = self._get_valid_service_url(self._inner.tsa_urls)
```

Comment:
For rekor urls, I think `get_valid_service_url()` needs updating to potentially support a major version 2

- https://github.com/sigstore/sigstore-python/blame/baa95acc60082ab37d22813890085d384c8a6d37/sigstore/_internal/trust.py#L344

Reply:
sure, agreed. Or maybe rather rekor and tsa cases should not use `_get_valid_service_url()` at all and should do their own things that actually return more meaningful results in their cases: both should return multiple results after all